apps/site/pages/courses/[courseSlug]/[lessonSlug]/index.tsx

```diff
@@ -87,9 +87,13 @@ function usePreferredMediaType(lesson: LessonProps["lesson"]) {
 	// Handle situations that content creator may created an empty lesson (to add content later)
 	const content = lesson.content as LessonContent;
 	const router = useRouter();
-	const [preferredMediaType, setPreferredMediaType] = useState(
-		content.length > 0 ? content[0].type : null
-	);
+	const [preferredMediaType, setPreferredMediaType] = useState<
+		"video" | "article" | "pdf" | "iframe"
+	>("video");
+
+	useEffect(() => {
+		setPreferredMediaType(content.length > 0 ? content[0].type : "video");
+	}, [content]);
```

Comment:
unschönes useEffect: https://react.dev/learn/you-might-not-need-an-effect

Wir haben hier keine performance probleme. Also kann man content auch direkt jedes mal ableiten:

```suggestion
const preferredMediaType = content.length > 0 ? content[0].type : "video"
```